tests/test_indexer.py

```diff
@@ -72,23 +74,44 @@ def test_process_new_indexable_object(self):
         # Better to write a search test method that would retry the search until
         # the expected result was acheived or a timeout was reached.
         time.sleep(5)
-        self._verify_search_results(1)
+        self.verify_search_results(1)
 
-    def _create_sample_s3_bundle_created_event(self, bundle_key: str) -> Dict:
+    @staticmethod
+    def create_sample_s3_bundle_created_event(bundle_key: str) -> Dict:
         with open(os.path.join(os.path.dirname(__file__), "sample_s3_bundle_created_event.json")) as fh:
             sample_s3_event = json.load(fh)
         sample_s3_event['Records'][0]["s3"]["bucket"]["name"] = utils.get_env("DSS_S3_TEST_BUCKET")
         sample_s3_event['Records'][0]["s3"]["object"]["key"] = bundle_key
         return sample_s3_event
 
-    def _verify_search_results(self, expectedHitCount):
+    def verify_search_results(self, expected_hit_count):
         es_client = Elasticsearch()
-        s = Search(using=es_client, index=HCA_ES_INDEX_NAME, doc_type=HCA_METADATA_DOC_TYPE).query("match", state="new")
-        response = s.execute()
-        self.assertEqual(expectedHitCount, len(response.hits))
+        query = "\
+            { \
+                \"query\": { \
+                    \"bool\": { \
+                        \"must\": [{ \
+                            \"match\": { \
+                                \"files.sample_json.donor.species\": \"Homo sapiens\" \
+                            } \
+                        }, { \
+                            \"match\": { \
+                                \"files.assay_json.single_cell.method\": \"Fluidigm C1\" \
+                            } \
+                        }, { \
+                            \"match\": { \
+                                \"files.sample_json.ncbi_biosample\": \"SAMN04303778\" \
+                            } \
+                        }] \
+                    } \
+                } \
+            }"
```

Comment:
Please rewrite this as a Python dict, instead of multiline string, and then call `es_client.search(body=json.dumps(query))`.